contracts/LPVault.sol

```diff
@@ -42,45 +62,50 @@ contract LPVault is Owned {
     // @dev An array that contains all the payments for this LPVault
     Payment[] public payments;
 
-    /// @dev `liquidPledging` is the only address that can call a function with
-    ///  this modifier
+    /// @dev The attached `LiquidPledging` contract is the only address that can
+    ///  call a function with this modifier
     modifier onlyLiquidPledging() {
         require(msg.sender == address(liquidPledging));
         _;
     }
-    /// @dev USED FOR TESTING???
+    /// @dev Used for testing... SHOULD BE REMOVED FOR MAINNET LAUNCH OR MAYBE NOT????????????????????....................
     function VaultMock() public pure {
 
     }
-
+    /// @dev The fall back function allows ETH to be deposited into the LPVault
+    ///  through a simple send
     function () public payable {
 
     }
 
-    /// @notice `setLiquidPledging` is used to attach a specific liquid pledging
-    ///  instance to this LPvault. Keep in mind this isn't a single pledge but
-    ///  instead an entire liquid pledging contract.
+    /// @notice `onlyOwner` used to attach a specific liquidPledging instance
+    ///  to this LPvault; keep in mind that once a liquidPledging contract is 
+    ///  attached it cannot be undone, this vault will be forever connected
     /// @param _newLiquidPledging A full liquid pledging contract
     function setLiquidPledging(address _newLiquidPledging) public onlyOwner {
         require(address(liquidPledging) == 0x0);
         liquidPledging = LiquidPledging(_newLiquidPledging);
+        VaultSet(_newLiquidPledging);
```

Comment:
why is this necessary? This will most always be the second call to the vault, so it doesn't seem like we need to log this

Reply:
It was a thought, I'm down to kill it. My thinking was that it is a pretty important piece and its something that we would want transparently declared, but since it can only be done once, I'll take it out.